src/backend/commands/dbcommands.c

```diff
@@ -1466,6 +1461,11 @@ createdb(ParseState *pstate, const CreatedbStmt *stmt)
 			CreateDatabaseUsingFileCopy(src_dboid, dboid, src_deftablespace,
 										dst_deftablespace);
 
+		/*
+		 * Update global last written LSN after wal-logging create database command
+		 */
+		SetLastWrittenLSNForDatabase(XactLastRecEnd);
```

Comment:
Yes, it is for cluster. I didn't see much sense in maintaining last written LSN for each database, because most accesses are per-relation and number of database can be quite larger, so we have to maintain hash for databases which complicates and slowdown code.

But I do not want to rename this function to `SetLastWrittenLSNForCluster()` mostly because word cluster is too overloaded. Yes, in context of Postgres it means set of databases. But still. But I you insist, then I will rename it.